### PR TITLE
Clean up front page stats on small devices

### PR DESCRIPTION
**Pull request by ericmj**
This is kind of hacky in some places but should improve things on small devices.

### CODE REVIEW

File: lib/hexpm/web/views/view_helpers.ex [hunk 1]
```diff
@@ -187,16 +187,39 @@ defmodule Hexpm.Web.ViewHelpers do
     text
   end
 
-  def human_number_space(string) when is_binary(string) do
-    split         = rem(byte_size(string), 3)
-    string        = :erlang.binary_to_list(string)
-    {first, rest} = Enum.split(string, split)
-    rest          = Enum.chunk(rest, 3) |> Enum.map(&[" ", &1])
-    IO.iodata_to_binary([first, rest])
+  @number_unit ["t", "m", "b"]
```
Comment (wojtekmach) [inline lib/hexpm/web/views/view_helpers.ex]: WDYT about 100K, 1M, 1B etc?

Comment (ericmj) [reply lib/hexpm/web/views/view_helpers.ex]: Yeah, that's better.

File: lib/hexpm/web/templates/page/index.html.eex [hunk 2]
```diff
@@ -62,46 +62,61 @@
     <li>
       <%= icon(:octicon, :package, class: "stat-icon stat-icon-package") %>
       <span class="stat-info">
-        <span class="count-info">
-          <%= human_number_space @num_packages %>
-        </span> <br>
-        packages <br> available
+        <span class="count-info wrap">
+          <%= human_number_space(5560, 5) %>
```
[the remark this answers is not in the document]
Comment (ericmj) [reply lib/hexpm/web/templates/page/index.html.eex]: Oops.

**Comment by ericmj** [comment]
> I'm gonna push couple commits to this branch if you want to continue that route, feel free to reset them back though and I'll continue there later on.

Yes, please!

**Pull request merged**